odoo/fields.py

```diff
@@ -1709,6 +1709,8 @@ def convert_to_write(self, value, record):
 class _String(Field):
     """ Abstract class for string fields. """
     translate = False                   # whether the field is translated
+    size = None                         # maximum size of values (deprecated)
+    allow_data = False
```

Comment:
What is this? Could you describe what is the purpose of this flag?
It looks like it is there to show a warning if a non-bytes/str is assigned to the field.